docs/css/footer.css

```diff
@@ -0,0 +1,63 @@
+#footer {
+
+}
+
+.footer-item {
+	border-bottom: 1px solid var(--color-form);
+	padding: var(--unit-space) var(--unit-gap);
+}
+
+.footer-item:last-child,
+.footer-sub-item:last-child {
+	border: 0 none;
+}
+
+.footer-item-link {
+	text-decoration: none;
+}
+
+.active .footer-item-link,
+.footer-item-link:hover {
+	text-decoration: underline;
+}
+
+#footer .active .footer-item-link {
```

Comment:
do you need the ID at this point? If not, maybe delete it - it's a high css specificity